### PR TITLE
Wrap strings in translation function| Patch #2

### PR DESCRIPTION
**Pull request by Drivingralle**
Some more changes to get #76 forward

### CODE REVIEW

**Comment by eddysims** [comment]
@Drivingralle I think this will be out of date now that your other PR is merged in, correct?

File: src/plugin/inc/settings-page.php [hunk 1]
```diff
@@ -16,8 +16,8 @@
 	 */
 	function find_my_blocks_register_options_page() {
 		add_management_page(
-			'Find My Blocks',
-			'Find My Blocks',
+			esc_html__( 'Find My Blocks', 'mytextdomain' ),
+			esc_html__( 'Find My Blocks', 'mytextdomain' ),
```
Comment (eddysims) [inline src/plugin/inc/settings-page.php:20]: Should these not be `'find-my-blocks'`?

**Review by eddysims — CHANGES_REQUESTED** [review]

File: src/plugin/inc/settings-page.php (hunk 1 above)
Comment (Drivingralle) [reply src/plugin/inc/settings-page.php]: True.

**Review by Drivingralle — COMMENTED** [review]
Fix wrong textdomain

**Pull request closed without merging**